Library/Formula/reposurgeon.rb

```diff
@@ -27,6 +35,18 @@ def install
     ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
     system "make", "install", "prefix=#{prefix}"
     (share/"emacs/site-lisp").install "reposurgeon-mode.el"
+
+    if build.with? "cython"
+      resource("cython").stage do
+        system "python", *Language::Python.setup_install_args(buildpath/"vendor")
+      end
+      ENV.prepend_path "PYTHONPATH", buildpath/"vendor/lib/python2.7/site-packages"
+      ohai "Building cyreposurgeon", "This may take a while"
```

Comment:
Let's not `ohai`. People can flip the `--verbose` switch if they worry about builds appearing to stall.